code/game/objects/items/ego_weapons/zayin.dm

```diff
@@ -58,6 +58,23 @@
 	armortype = WHITE_DAMAGE
 	attack_verb_continuous = list("smacks", "strikes", "beats")
 	attack_verb_simple = list("smack", "strike", "beat")
+	var/pulse_cooldown
+	var/pulse_cooldown_time = 1 SECONDS
+	var/pulse_damage = -0.5
+
+/obj/item/ego_weapon/penitence/equipped(mob/user, slot = ITEM_SLOT_HANDS, initial = FALSE)
```

Comment:
why is this here? seems unrelated to the PR

Reply:
I forgot to delete that, hold on